src/figuresClasses.ts

```diff
@@ -1,19 +1,97 @@
-export interface Figure {
 
+type Color = 'red' | 'green' | 'blue';
+type Shape = 'triangle' | 'circle' | 'rectangle';
+
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea(): number;
 }
 
-export class Triangle {
+let p: number;
+let square: number;
```

Comment:
You don't need these variables in a global scope. You have different formulas for every figure, so use const inside getArea() methods